fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -5323,6 +5348,31 @@
 | `in=(null)`| (initial)| `<null>` |
 | `in=(undefined)`| (initial)| `<null>` |
 
+## `inert` (on `<div>` inside `<div>`)
```

Comment:
Shows that you should use `inert={insert ? "true" ? false}` at the moment to ensure forwards compatibility at the cost of having a warning in current and future versions.